examples/query-suggestions-with-hits/types/ProductHit.ts

```diff
@@ -0,0 +1,18 @@
+import { Hit } from '@algolia/client-search';
+
+type ProductRecord = {
+  brand: string;
+  categories: string[];
+  description: string;
+  image: string;
+  name: string;
+  price: number;
+  rating: number;
```

Comment:
Perhaps we can add the `url` type here too:

```suggestion
  rating: number;
  url: string;
```

Reply:
Make sense, added in https://github.com/algolia/autocomplete/pull/532/commits/a6cde304fa8f6d0d15093dffec1e844c98bbed72